TTS_xwing/src/AiSheetDb.ttslua

```diff
@@ -0,0 +1,194 @@
+masterAiSheetDb={}
```

Comment:
I attempted to get AI Sheet spawning through the data pad and got most of the way there before realizing the actual spawning code would require a ton of work.  I removed my changes to the data pad but kept this file in the event that myself or someone else wants to make a way to dynamically spawn the AI Sheets (instead of just having them in bags on the table).